syscalls.c

```diff
@@ -290,14 +296,15 @@ int syscalls_gettid(void *ustack)
 
 int syscalls_beginthreadex(void *ustack)
 {
+	typedef void (*start_func)(void *harg);
```

Comment:
is this the best place for typedef? Maybe the arg to `proc_threadCreate` should be strongly-typed (with type definition near function declaration?)